cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -490,18 +548,25 @@ def act_sigmoid_scaled(x):
     else:
         K.set_session(tf.Session(config=tf.ConfigProto(device_count={"GPU": 0})))
 
-    # build the model
+    # Model compiling
+    # ================
+    #
```

Comment:
Please remove these lines. We shouldn't be having any headings in the middle of a function.